assets/scss/2-layout/2-navigation.scss

```diff
@@ -0,0 +1,31 @@
+nav {
+  margin-top: 30px;
+  display: none;
+
+  @media all and (min-width: 600px) {
```

Comment:
Looks like you're using `600px` breakpoint quite a bit. Consider extracting them to variables ;)

```
$breakpoint-mobile-up: 600px;
$breakpoint-mobile-down 599px;
```